<launch>
  <include file="$(find pr2_grasp_behaviors)/launch/simple_grasp_pipeline.launch"/>
  <include file="$(find pr2_playpen)/launch_pr2/UI_simple_grasp_pipeline.launch"/>

  <node pkg="pr2_collision_monitor" type="collision_monitor" name="l_collision_monitor" output="screen" >
    <param name="arm" type="str" value="l"/>
    <param name="training_mode" type="bool" value="false"/>
    <param name="significance_mode" type="bool" value="true"/>
    <param name="behavior_name" type="str" value="overhead_grasp"/>
    <param name="filename_prefix" type="str" value="$(find pr2_grasp_behaviors)/data/"/>
    <rosparam param="filename_list">
        [overhead_grasping_l_data.bag]
    </rosparam>

  </node>                 

  <node pkg="pr2_collision_monitor" name="l_arm_moving_server" type="arm_moving_server" output="screen">
    <param name="arm" type="str" value="l"/>
  </node>

  <node pkg="pr2_grasp_behaviors" name="l_overhead_grasp_behavior" type="overhead_grasp_behavior.py" args="l server" output="screen"/>

</launch>
